contrib/meson/meson/lib/meson.build

```diff
@@ -43,7 +43,7 @@ liblz4_dep = declare_dependency(
   include_directories: include_directories(lz4_source_root / 'lib')
 )
 
-if get_option('tests') or get_option('programs')
+if get_option('tests') or get_option('programs') or get_option('programs')
```

Comment:
`get_option('programs')` seems repeated twice

Reply:
I commited it too early. I didn't think you're going to review it right away but I guess you got a few mails :/